src/routes/safe/components/Apps/hooks/useAppsSearch.ts

```diff
@@ -0,0 +1,25 @@
+import { useMemo } from 'react'
+import Fuse from 'fuse.js'
+import { SafeApp } from 'src/routes/safe/components/Apps/types'
+
+const useAppsSearch = (apps: SafeApp[], searchText: string): SafeApp[] => {
+  const fuse = useMemo(
+    () =>
+      new Fuse(apps, {
+        keys: ['name', 'description'],
```

Comment:
Can we somehow prioritize the name over description? I.e. sort the name matches to the beginning?